cpmonitor/tests/cleanupimages_test.py

```diff
@@ -63,6 +43,7 @@ def tearDown(self):
         self.assertFalse(os.path.isdir(self.uploads_path))
         self.assertTrue(os.path.isdir(self.base_path))
 
+    @override_settings(BASE_DIR=base_path, MEDIA_ROOT=images_path)
```

Comment:
I think you could also annotate the whole test class with this.

Reply:
Unfortunately not because the `base_path` and `images_path` are class variables and then out of scope